openfl/component/aggregator/aggregator.py

```diff
@@ -973,82 +960,17 @@ def _prepare_trained(self, tensor_name, origin, round_number, report, agg_result
             report (bool): Whether to report.
             agg_results (np.array): Aggregated results.
         """
-        # The aggregated tensorkey tags should have the form of
-        # 'trained' or 'trained.lossy_decompressed'
-        # They need to be relabeled to 'aggregated' and
-        # reinserted. Then delta performed, compressed, etc.
-        # then reinserted to TensorDB with 'model' tag
-
-        # First insert the aggregated model layer with the
-        # correct tensorkey
+        # The aggregated tensorkey tags should have the form of 'trained' or
+        # 'trained.lossy_decompressed' they need to be relabeled to 'aggregated' and reinserted.
+
+        # First insert the aggregated model layer with the correct tensorkey
         agg_tag_tk = TensorKey(tensor_name, origin, round_number + 1, report, ("aggregated",))
         self.tensor_db.cache_tensor({agg_tag_tk: agg_results})
 
-        # Create delta and save it in TensorDB
```

Comment:
This block should not be deleted entirely. If lossy compression is used and/or deltas are calculated between models, I expect this will lead to problems. I understand removing this results in significant memory savings. I would suggest making this entire block conditional for now to reduce memory for existing examples / SFAI.